c2corg_api/views/user.py

```diff
@@ -288,6 +296,146 @@ def post(self):
         return {}
 
 
+class UpdateAccountSchema(colander.MappingSchema):
+    email = colander.SchemaNode(
+            colander.String(),
+            missing=colander.drop,
+            validator=colander.All(
+                colander.Email(),
+                colander.Function(
+                    partial(available_user_attribute, 'email'),
+                    'Already used email'
+                )
+            ))
+    toponame = colander.SchemaNode(
```

Comment:
Can we use consistent names for these fields? `name` instead of `toponame` and `forum_name` instead of `forumname`.